features/native_crash_handling.feature

```diff
@@ -1,212 +1,5 @@
 Feature: Native crash reporting
 
-    Scenario: Dereference a null pointer
```

Comment:
Did you mean to remove all of these?

Reply:
Nope, that was accidentally left in after speeding up a run of two scenarios with BrowserStack local. I've reinstated these test cases